tests/ci.sh

```diff
@@ -5,10 +5,45 @@
 
 set -xeuf -o pipefail
 
+error_msg () {
+    echo "Error: $1"
+    usage
```

Comment:
Is usage some function?

Reply:
Negative use case shows the following log,
```
+ MISMATCHER=
+ '[' 1 -gt 0 ']'
+ case "$1" in
+ error_msg 'Unknown argument: mismatch'
+ echo 'Error: Unknown argument: mismatch'
Error: Unknown argument: mismatch
+ usage
./tests/ci.sh: line 10: usage: command not found
```

Reply:
Done